### DISCUSSION TITLE
How to hide the blocking message

### DISCUSSION THREAD

**Discussion by olaf-ej** [HomeITAdmin/nextcloud_geoblocker#108] [Ideas]
Hi,
perhaps I'm paranoïd, but I think blocking message is a good information for a hacker.
I explain: if I'm a Chinese hacker, and trying to connect randomly to a nextcloud/geoblocker site, I can see immediatly that it's a french site, and use a VPN to try to connect again.
So my question: is it possible to prevent geoblocker to display the blocking message ? 
Or how to personnalize message so you could keep the term "Geoblocker" in the blocking page. 
Or how to change the language of this page. Because English, or spanish is widely use in the world.
Kind regards.

**Comment by HomeITAdmin** [comment]
Hi, in this issue we had the same "discussion": https://github.com/HomeITAdmin/nextcloud_geoblocker/issues/99
Bottom line is, the use of the Geoblocker app is very limited independent of the message shown. 

You maybe get some benefit in the direction you are thinking with geoblocking on network level. This you would need to configure outside Nextcloud.
